log_outgoing_requests/handlers.py

```diff
@@ -40,17 +49,10 @@ class DatabaseOutgoingRequestsHandler(logging.Handler):
     If any of the conditions don't match, then the body is omitted.
     """
 
-    def emit(self, record: AnyLogRecord):
-        from .models import OutgoingRequestsLog, OutgoingRequestsLogConfig
+    def log_request(self, record, config):
```

Comment:
can you please use/stick to the existing type hints? There should be sufficient code to take inspiration from.